zetaclient/chains/evm/observer/v2_inbound_tracker.go

```diff
@@ -28,7 +33,7 @@ func (ob *Observer) ProcessInboundTrackerV2(
 	}
 
 	for _, log := range receipt.Logs {
-		if log == nil {
+		if log == nil && log.Address != gatewayAddr {
```

Comment:
Is this a typo?

Reply:
Good catch, they should make the linter catching this

Reply:
https://github.com/zeta-chain/node/pull/3226/commits/c00f9644a73f30771a74adeaa15fd9a6ffbdef40